### DISCUSSION TITLE
We should archive the discord server and move all discussion here

### DISCUSSION THREAD

**Discussion by Adyel** [OpenBangla/OpenBangla-Keyboard#188] [General]
GitHub discussion is pretty nice. I think we don't need any 3rd party chat application anymore. We should close the discord server and suggest people to ask their questions here. What do you guys think?

@mominul @bdeshi

**Comment by mominul** [comment]
I also agree!
We'll need to leave an announcement there that we have moved and update the README.md as well.

**Reply by Adyel** [reply]
@mominul Yup. I have disabled creating any new post on the discord server. Members can still view them. We should put an announcement on the channels and change the README on GitHub.
